gtk/transmission-gtk.metainfo.xml.in

```diff
@@ -29,6 +29,14 @@ Copyright 2017 Endless Mobile, Inc.
     </p>
   </description>
   <url type="homepage">https://transmissionbt.com/</url>
+  <url type="bugtracker">https://github.com/transmission/transmission/issues</url>
+  <url type="faq">https://transmissionbt.com/help/gtk/4.0x/html/FAQ.html</url>
+  <url type="help">https://transmissionbt.com/help/gtk/4.0x/index.html</url>
```

Comment:
These links and the "releases" section below add to the list of things to keep track of on every release. I'm worried they'll get stale, unless there's some sort of automation or at least a checklist item.

Reply:
we could make a version-independent landing page that points to the various versions' faq / help

Reply:
Where would be the right place to add a checklist item or automation for the `<releases>` section?

Reply:
`appstreamcli` does have a `news-to-metainfo` command that tries to parse a text-based NEWS file and emit appstream XML. Sadly it can't understand `news/news-4.0.3.md`.

(The other way around is also supported: generating text-based NEWS files from the metainfo.)